BitmovinComScoreAnalytics.podspec

```diff
@@ -6,6 +6,7 @@ Pod::Spec.new do |s|
   Comscore Analytics Integration for the Bitmovin Player iOS SDK
                        DESC
 
+  s.swift_version    = '4.0'
```

Comment:
Why not using latest swift version `4.2`?

Reply:
confirmed Turner was okay moving to 4.2